src/applications/personalization/profile/components/proof-of-veteran-status/ProofOfVeteranStatus.jsx

```diff
@@ -1,52 +1,145 @@
+    /android/i.test(userAgent);
+
+  const eligibilityMap = militaryInformation.serviceHistory.serviceHistory.map(
+    service =>
+      service.characterOfDischargeCode
+        ? DISCHARGE_CODE_MAP[service.characterOfDischargeCode].indicator
+        : 'N',
+  );
+
+  const pdfData = {
+    title: `Veteran status card for ${formatFullName({
+      first,
+      middle,
+      last,
+      suffix,
+    })}`,
+    details: {
+      fullName: formatFullName({ first, middle, last, suffix }),
+      serviceHistory: militaryInformation.serviceHistory.serviceHistory.map(
+        item => {
+          return {
+            ...item,
+            branchOfService: getServiceBranchDisplayName(item.branchOfService),
+          };
+        },
+      ),
+      totalDisabilityRating,
+      dob: renderDOB(dob),
+      image: {
+        title: 'V-A logo',
+        url: 'https://www.va.gov/img/design/logo/logo-black-and-white.png',
+      },
+    },
+  };
+
+  const createPdf = () => {
+    recordEvent({
+      event: 'file_download',
+      'click-url': PROFILE_PATHS.MILITARY_INFORMATION,
+      'file-name': 'Veteran status card',
+      'file-extension': 'pdf',
+    });
+
+    generatePdf('veteranStatus', 'Veteran status card', pdfData, !isMobile);
+  };
 
-const ProofOfVeteranStatus = () => {
   return (
-    <div id="proof-of-veteran-status">
-      <h2 className="vads-u-font-size--h3 vads-u-margin-top--4 vads-u-margin-bottom--1p5">
-        Proof of Veteran status
-      </h2>
-      <p>
-        You can use your Veteran status card to get discounts offered to
-        Veterans at many restaurants, hotels, stores, and other businesses.
-      </p>
-      <p>
-        <strong>Note: </strong>
-        This card doesn’t entitle you to any VA benefits.
-      </p>
-      <div className="vads-u-margin--1 vads-u-margin-y--2">
-        <img
-          width="400rem"
-          src="/img/content/posts/proof-of-veteran-status-card-example.png"
-          alt="sample proof of veteran status card featuring name, date of birth, disability rating and period of service"
-        />
-      </div>
-      <div className="vads-u-font-size--md">
-        <i className="fa fa-app-store" />{' '}
-        <va-link
-          role="link"
-          download
-          filetype="PDF"
-          href={null}
-          text="Download and print your Veteran status card"
-          onClick={() => {}}
-        />
-      </div>
-      <div className="vads-u-margin-y--4">
-        <MobileAppCallout
-          headingText="Get proof of Veteran Status on your mobile device"
-          bodyText={
-            <>
-              You can use our mobile app to get proof of Veteran status. To get
-              started, download the <strong> VA: Health and Benefits </strong>{' '}
-              mobile app.
-            </>
-          }
-        />
-      </div>
-    </div>
+    <>
+      {militaryInformation.serviceHistory.serviceHistory.length > 0 &&
+      eligibilityMap.includes('Y') ? (
```

Comment:
Suggestion: You don't need a ternary with  `: null` and should be able to instead use `&& ( render stuff )`

Reply:
Our goal is to add some eligibility messaging to the user to explain why they might not be seeing the Veteran Status download (i.e. they have a service history but the `characterOfDischargeCode` is one of the "unknown" codes for example) so if it's ok with you as is I'll leave the ternary for now so it's set up for an alternate.